src/run.py

```diff
@@ -116,15 +120,15 @@ def main():
 
     # Print the structure of the book
     print('Structure of the book:')
-    structure, _ = book.get_structure()
-    print(structure)
+    structure = book.get_structure()
+    print(book)
```

Comment:
this seems like unwanted change?

Reply:
This is actually how I'm using it right now and there are some changes to make testing a bit easier but I'm only have a problem getting the structure and yeah I changed it back to print structure instead of print book https://replit.com/@CJSTRYKER/AuthorGPT